### PR TITLE
Shows how to download photos with batching and prepare them for rendering in a browser

### PR DESCRIPTION
**Pull request by plamber**
## Summary
This example shows how to download multiple photos with batching and how to convert them in base64 for further rendering. Useful for people picker scenarios.

## Closing issues
Closes: #367

## Checklist

-   [x] I have read the **CONTRIBUTING** document.
-   [x] My code follows the code style of this project.
-   [x] My change requires a change to the documentation.
-   [x] I have updated the documentation accordingly.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

### CODE REVIEW

File: docs/content/Batching.md [hunk 1]
```diff
@@ -66,6 +66,81 @@ const serialBatching = async function(elem) {
 };
 ```
 
+### Download multiple profile photos with batching and prepare manipulate these for rendering in a browser
```
Comment (baywet) [inline docs/content/Batching.md:69]:
```suggestion
### Download multiple profile photos with batching and preprocess these for rendering in a browser 
```

File: docs/content/Batching.md [hunk 2]
```diff
@@ -66,6 +66,81 @@ const serialBatching = async function(elem) {
...
+      // initiate the batchrequest and execute the operation
+      const batchRequestContent = new BatchRequestContent(batchRequestSteps);
+      const content = await batchRequestContent.getContent();
+      const batchResponse = new BatchResponseContent(await client.api("/$batch").post(content));
+  
+      // example on how to retrieve the base64 representation of the downloaded image for the first user
+      const response = batchResponse.getResponseById(users[0]);
+      if (response.ok) {
+        var data = await response.text();
+        const _binToBlob = await this.b64toBlob((data),'img/jpg');
```
Comment (baywet) [inline docs/content/Batching.md:132]:
```suggestion
        const binToBlob = await this.b64toBlob((data),'img/jpg');
```

File: docs/content/Batching.md [hunk 3]
```diff
@@ -66,6 +66,81 @@ const serialBatching = async function(elem) {
...
+        
+        // you can associate the base64 output to an src attribute of an <img> HTML tag
+        const base64Result = await this.blobToBase64(_binToBlob);
```
Comment (baywet) [inline docs/content/Batching.md:135]:
```suggestion
        const base64Result = await this.blobToBase64(binToBlob);
```

**Review by baywet — CHANGES_REQUESTED** [review]
Great addition! Thanks for putting it together so fast. I added a few minor comments for consistency and word use

**Review by nikithauc — DISMISSED** [review]

**Review by nikithauc — APPROVED** [review]

**Review by baywet — APPROVED** [review]

**Pull request merged**